.github/workflows/tests_cpu_versions.yml

```diff
@@ -0,0 +1,33 @@
+on: [pull_request]
+
+jobs:
+  build-cpu-versions:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
+        python-version: [3.6, 3.7, 3.8]
+        pytorch-version: [1.5.1, 1.6.0]
```

Comment:
@shijianjian I detected some fails against 1.5.1 in augmentations that I think that can be fixed.

Reply:
Isn't that irrelevant here? According to 

https://github.com/kornia/kornia/blob/339340b4a61153edcc2cb1467ba035c93c878c3f/setup.py#L27-L35

as of `kornia==0.4`, `torch==1.5` is no longer supported.

Reply:
@pmeier can be. It\s kind of messy the versioning right now. not sure what could be the proper way to handle it

Reply:
Could you elaborate how it could be relevant? As is, the requirements state that the current version of `kornia` require `torch>=1.6`. Thus, `torch==1.5` is not officially supported and there is no need test for its compatibility.

Reply:
I suspect we didn't check well before

Reply:
In that case you (`kornia` core team) should decide if you want to support `torch==1.5`. If you want to, you are correct in checking this.

Reply:
/cc @shijianjian @ducha-aiki

Reply:
@pmeier we are considering to have better backward compatibility. After this PR, we will try to enable 1.5.1 first.